static/js/stats/stats.js

```diff
@@ -1034,10 +1034,10 @@ function populate_messages_read_over_time(data) {
     }
 }
 
-function get_chart_data(data, callback) {
+function get_chart_data(opts, callback) {
     $.get({
         url: "/json/analytics/chart_data" + page_params.data_url_suffix,
-        data,
+        data: opts,
         idempotent: true,
         success(data) {
```

Comment:
Yeah, seeing this pattern repeat, I increasingly like the idea of changing the `success` handler variable names to `response` or `response_data`, since it avoids introducing this duplicate `opts` name that ends up being less clean since we can't use destructuring syntax.